internal/gapicgen/generator/config.go

```diff
@@ -770,6 +770,14 @@ var microgenGapicConfigs = []*microgenConfig{
 		apiServiceConfigPath:  "google/cloud/functions/v1/functions_gapic.yaml",
 		releaseLevel:          "ga",
 	},
+	{
+		inputDirectoryPath:    "google/cloud/networkconnectivity/v1alpha1",
+		pkg:                   "networkconnectivity",
+		importPath:            "cloud.google.com/go/networkconnectivity/v1alpha1",
```

Comment:
```suggestion
		importPath:            "cloud.google.com/go/networkconnectivity/apiv1alpha1",
```

Reply:
Looks like we might need to upgrade our instructions, but at least this failed a unit test! The version should start with `api`. This was done as to not clash with Go's module version system where a folder named something like `v1` has meaning to Go tooling. After updating the code will need to be regenerated.